internal/tools/securitypolicy/helpers/helpers.go

```diff
@@ -137,18 +142,123 @@ func PolicyContainersFromConfigs(containerConfigs []securitypolicy.ContainerConf
 			workingDir = containerConfig.WorkingDir
 		}
 
-		container, err := securitypolicy.NewContainer(
+		container, err := CreateContainerPolicy(
 			containerConfig.Command,
 			layerHashes,
 			envRules,
 			workingDir,
 			containerConfig.ExpectedMounts,
+			containerConfig.Mounts,
 		)
 		if err != nil {
 			return nil, err
 		}
-		policyContainers = append(policyContainers, container)
+		policyContainers = append(policyContainers, &container)
 	}
 
 	return policyContainers, nil
 }
+
+// CreateContainerPolicy creates a new Container policy instance from the
+// provided constraints or an error if parameter validation fails.
+func CreateContainerPolicy(
+	command, layers []string,
+	envRuleConfigs []config.EnvRuleConfig,
+	workingDir string,
+	eMounts []string,
+	mntConfigs []config.MountConfig,
+) (securitypolicy.Container, error) {
+	envRules, err := createEnvVarRules(envRuleConfigs)
+	if err != nil {
+		return securitypolicy.Container{}, err
+	}
+	mntConstraints, err := createMountConstraints(mntConfigs)
+	if err != nil {
+		return securitypolicy.Container{}, err
+	}
+	return securitypolicy.NewContainer(
+		command,
+		layers,
+		envRules,
+		workingDir,
+		eMounts,
+		mntConstraints,
+	), nil
+}
+
+func createEnvVarRules(rules []config.EnvRuleConfig) ([]securitypolicy.EnvVarRule, error) {
+	var envRules []securitypolicy.EnvVarRule
+	for _, rule := range rules {
+		switch rule.Strategy {
+		case securitypolicy.EnvRuleRegex:
```

Comment:
This is currently not implemented right since `NewEnvVarRuleConfigs` does not allow for it?

Reply:
sorry, why do you think `NewEnvVarRuleConfigs` doesn't allow this?